homeassistant/components/fan/xiaomi_miio.py

```diff
@@ -48,7 +49,8 @@
          'zhimi.airpurifier.v5',
          'zhimi.airpurifier.v6',
          'zhimi.humidifier.v1',
-         'zhimi.humidifier.ca1']),
+         'zhimi.humidifier.ca1',
+         'zhimi.airfresh.va2']),
```

Comment:
In the future you may want to like to keep the `])` in a separate line (to avoid two-line diffs when adding models).